Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM opensciencegrid/software-base:fresh
+
+# Install dependencies (application, Apache)
+RUN \
+    yum update -y && \
+    yum install -y \
+      gcc \
+      git \
+      libyaml-devel \
+      mod_ssl \
+      python3-devel \
+      python3-ldap \
+      python3-pip \
+    yum install -y \
```

Comment:
```suggestion
    && yum install -y \
```

Reply:
Good catch on the missing separator. I'll get that fixed!